challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map do |n| n.capitalize end
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  return input.select do |n| n == "dog" end
```

Comment:
Rubyists generally prefer implicitly `return`ing the final value from a method.

https://github.com/bbatsov/ruby-style-guide#no-explicit-return